package/MDAnalysis/analysis/hbonds/hbond_autocorrel.py

```diff
@@ -392,12 +392,10 @@ def run(self, force=False):
         # for normalising later
         counter = np.zeros_like(master_results, dtype=np.float32)
 
-        pm = ProgressMeter(self.nruns, interval=1,
-                           format="Performing run %(step)5d/%(numsteps)d"
-                                  "[%(percentage)5.1f%%]")
-
-        for i, (start, stop) in enumerate(zip(self._starts, self._stops)):
-            pm.echo(i)
+        n_frames = len(self.u.trajectory)
```

Comment:
`n_frames` is the wrong total, should be `self.nruns`

Reply:
Oh yes ! my bad